Chargebee/Classes/Restore/CBRestorePurchaseManager.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CBRestorePurchaseManager.swift
+//  Chargebee
+//
+//  Created by ramesh_g on 20/02/23.
+//
+
+import Foundation
+import StoreKit
+
+public typealias CBRestorePurchaseHandler = (CBResult<CBRestorePurchase>) -> Void
+
+// MARK: - CBRestorePurchase
+public struct CBRestorePurchase: Decodable {
+    let inAppSubscriptions: [InAppSubscription]
+
+    enum CodingKeys: String, CodingKey {
+        case inAppSubscriptions = "in_app_subscriptions"
+    }
+}
+
+// MARK: - InAppSubscription
+public struct InAppSubscription: Decodable {
+    public let subscriptionID: String?
+    public let planID: String?
+    public let storeStatus: String?
```

Comment:
- Can we make `storeStatus` as `enum`?
- Can we make `planID` and `subscriptionID` as non-optional?

Reply:
Yes updated storeStatus as enum and  planID and subscriptionID as non-optionals